test/integration/util/util.go

```diff
@@ -120,7 +122,7 @@ type OcRunner struct {
 }
 
 func NewOcRunner() *OcRunner {
-	p, _ := exec.LookPath("oc")
+	p := path.Join(os.Getenv(constants.MiniShiftHomeEnv), "cache", "oc", "v1.4.1", "oc")
```

Comment:
Not exactly what I thought. What if I had to use this path in any other code? 

I think let's not do this as part of it. We can refactor it later or discuss the proper solution.